lib/icingadb/redisconnection.hpp

```diff
@@ -62,8 +62,9 @@ namespace icinga
 		enum class QueryPriority : unsigned char
 		{
 			Heartbeat,
-			Config,
-			State,
+			RuntimeStateStream, // runtime state update notifications, doesn't affect initially synced states
```

Comment:
I'd say `notifications` isn't correct in this case, because we do not only send a notification here, but the whole state.